Rehlate Landing Page/index.html

```diff
@@ -0,0 +1,644 @@
+                    <div class="section-item-img p-relative">
+                       <img src="./assets/images/riyadh-img.jpg" alt="riyadh-img "/>
+                        <div class="section-travel-content primaryColor-bg    p-absolute bottom-30 ">
+                          
+                           <div class="container">
+                            <h4 class="darkBlack">الرياض</h4>
+                            <p class="Gray-100">2 أيام بتكلفة 1,500 دولار</p>
+                           </div>
+                          
+                        </div>
+                    </div>
+                </div>
+                     
+            </div>
+        </div>
+    </section>
+    <section id="Trips" class="trips">
+        <div class="container text-center">
+            <p class="GrayColor">تمتع في رحلتك</p>
+            <h2 class="darkBlack">مجموعة من خدماتنا</h2>
+            <div class="d-flex trip padding-top-100 padding-bottom-100  text-center radius-25 CandyGrape-100-bg gap-25">
+                
+                    
+                <div class="section-trips">
+                  
+                    <div class="section-trip-img">
+                        <img src="./assets/images/services-comp.png" alt="Tourguide"/>
+                    </div>
+                    <div class="section-trips-content">
+                        <h4>دليل سياحي</h4>
+                        <p>نقدم لك خدمات الإرشاد السياحي لبناء
+                            خطط سفر مميزة.</p>
+                    </div>
+                </div>
+                <div class="section-trips ">
+                    
+                    <div class="section-trip-img">
+                        <img src="./assets/images/services-car.png" alt="Tourguide"/>
+                    </div>
+                    <div class="section-trips-content">
+                        <h4>حجوزات فندقية</h4>
+                        <p>لا تقلق حول مكان إقامتك فنحن نهتم
+                            بكامل الحجوزات الخاصة بك.</p>
+                    </div>
+                </div>
+                <div class="section-trips">
+                    <div class="section-trip-img">
+                        <img src="./assets/images/services-cart.png" alt="Tourguide"/>
+                    </div>
+                    <div class="section-trips-content">
+                        <h4>رحلات ميدانية</h4>
+                        <p>لن تكون إجازتك مملة معنا فهنا الكثير
+                            من الخطط والرحلات المختلفة.</p>
+                    </div>
+                </div>
+            </div>
+        </div>  
+    </section>
+    <section id="Prices" class="prices">
+        <div class="container">
+            <p class="GrayColor">لا تقلق حول التكلفة</p>
+            <h2 class="darkBlack">خطط تناسب ميزانيتك</h2>
+            <div class="d-flex trip gap-25  ">
+                <div class="section-trip-price RiseNShine300-bg radius-25">
+                    <div class="section-trip-benefits">
```

Comment:
The first card in this section looks good, but the content of the other two cards is the same as the first. Please make sure the text follows the Figma design for all cards

Reply:
i changed the remaining of the two content done .